.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
+  "globals": {
+    "__DEV__": true
+  },
+  "env": {
+    "browser": true,
+    "mocha": true
+  },
+  "rules": {
+    "no-confusing-arrow": 0,
+    "react/jsx-quotes": 0,
+    "react/jsx-filename-extension": 0,
```

Comment:
According to https://github.com/kriasoft/react-starter-kit/issues/52, it is recommended to use the .jsx file extension. What do you think?